src/main/groovy/lang/ObjectRange.java

```diff
@@ -338,22 +337,22 @@ public int size() {
 
         // Performance detail:
         // not using get(fromIndex), get(toIndex) in the following to avoid stepping over elements twice
-        final Iterator<Comparable> iter = new StepIterator(this, 1);
+        final Iterator<Comparable> it = new LazySteppingIterator(this, 1);
```

Comment:
I notices that they were inconsistent so I went one way. Thanks for noticing, renamed all to `iter`.